mapping.ts

```diff
@@ -7,87 +7,87 @@
 export default {
   icons: {
     mdiAccount: 'fas-user',
-    mdiAccountCircleOutline: 'fas-user-circle',
+    mdiAccountCircleOutline: 'far-user-circle',
     mdiAccountGroup: 'fas-users',
     mdiAccountPlus: 'fas-user-plus',
-    mdiAlertCircleOutline: 'fas-exclamation-circle',
-    mdiAlertOutline: 'fas-exclamation-triangle',
-    mdiArrowDown: 'fas-arrow-down',
-    mdiArrowLeft: 'fas-arrow-left',
-    mdiArrowRight: 'fas-arrow-right',
-    mdiArrowUp: 'fas-arrow-up',
-    mdiBackupRestore: 'fas-undo',
-    mdiCalendarRange: 'fas-calendar-alt',
-    mdiChartPie: 'fas-pie-chart',
-    mdiCheck: 'fas-check',
+    mdiAlertCircleOutline: 'fasr-exclamation-circle',
+    mdiAlertOutline: 'fasr-exclamation-triangle',
+    mdiArrowDown: 'fasr-arrow-down',
+    mdiArrowLeft: 'fasr-arrow-left',
+    mdiArrowRight: 'fasr-arrow-right',
+    mdiArrowUp: 'fasr-arrow-up',
+    mdiBackupRestore: 'fass-clock-rotate-left',
+    mdiCalendarRange: 'far-calendar-range',
+    mdiChartPie: 'fass-pie-chart',
+    mdiCheck: 'fasr-check',
     mdiCheckBold: 'here is no equivalent in fontawesome',
-    mdiChevronDown: 'fas-chevron-down',
-    mdiChevronRight: 'fas-chevron-right',
-    mdiCircleOutline: null,
-    mdiCircleSlice3: null,
+    mdiChevronDown: 'fasr-chevron-down',
+    mdiChevronRight: 'fasr-chevron-right',
+    mdiCircleOutline: 'far-circle',
+    mdiCircleSlice3: 'far-circle-quarter-stroke',
     mdiClipboard: 'fas-clipboard',
     mdiClipboardFlow: 'fas-copy',
-    mdiClose: 'fas-close',
+    mdiClose: 'fasr-close',
     mdiCloudCheckVariant: 'fas-cloud-check',
-    mdiCog: 'fas-cog',
-    mdiContacts: 'fas-address-book',
+    mdiCog: 'fass-cog',
+    mdiContacts: 'fas-image-user',
     mdiDelete: 'fas-trash',
-    mdiDotsHorizontal: 'fas-ellipsis-h',
-    mdiDownload: 'fas-download',
+    mdiDotsHorizontal: 'far-ellipsis',
+    mdiDownload: 'fass-down-to-line',
     mdiDragVertical: 'fas-grip-lines-vertical',
     mdiEmail: 'fas-envelope',
-    mdiExclamation: 'fas-exclamation',
-    mdiEye: 'fas-eye',
-    mdiEyeOutline: null,
+    mdiExclamation: 'fasr-exclamation',
+    mdiEye: 'fass-eye',
+    mdiEyeOutline: 'fasr-eye',
     mdiFacebook: 'fab-facebook',
     mdiFile: 'fas-file',
-    mdiFileCompare: null,
-    mdiFileMultiple: null,
-    mdiFileUpload: 'fas-arrow-up',
+    mdiFileCompare: 'fas-file-plus-minus',
+    mdiFileMultiple: 'fas-files',
+    mdiFileUpload: 'fas-file-arrow-up',
     mdiFilter: 'fas-filter',
     mdiFolder: 'fas-folder',
     mdiFolderOpen: 'fas-folder-open',
-    mdiFormatListBulletedSquare: 'fas-list-ul',
-    mdiFullscreen: 'fas-maximize',
-    mdiFullscreenExit: 'fas-minimize',
+    mdiFormatListBulletedSquare: 'fasr-list',
+    mdiFullscreen: 'fass-expand',
+    mdiFullscreenExit: 'fass-compress',
     mdiImageEdit: null,
```

Comment:
This one is still `null`.

Reply:
it's a demo to show a fallback to MDI icons in case there is no similar asset in FA icons

Reply:
I saw it, but it was not obvious that this was a demo since it was not added by a commit marked with `[DEMO]`.